### PR TITLE
Module Contract tables and columns renaming

### PR DESCRIPTION
**Pull request by metas-rc**

### CODE REVIEW

**Review by adi-stefan — DISMISSED** [review]
Lgtm, but I just remembered: did we decide on keeping the (harvest) Year on the Settings level, or did we decide to move it to an intermediate entity, PLV style?

File: backend/de.metas.adempiere.adempiere/base/src/main/java-gen/org/compiere/model/X_Contract_Module_Log.java [hunk 1]
```diff
@@ -1,246 +0,0 @@
-// Generated Model - DO NOT CHANGE
```
Comment (adi-stefan) [inline backend/de.metas.adempiere.adempiere/base/src/main/java-gen/org/compiere/model/X_Contract_Module_Log.java:1]: Shouldn't you also remove the interfaces as well?

**Review by adi-stefan — CHANGES_REQUESTED** [review]
Mostly lgtm, but I just remembered: did we decide on keeping the (harvest) Year on the Settings level, or did we decide to move it to an intermediate entity, PLV style?

File: backend/de.metas.adempiere.adempiere/base/src/main/java-gen/org/compiere/model/X_Contract_Module_Log.java (hunk 1 above)
Comment (metas-ts) [reply backend/de.metas.adempiere.adempiere/base/src/main/java-gen/org/compiere/model/X_Contract_Module_Log.java:1]: yes. lemmi check, if was thinking i did that

**Comment by metas-ts** [comment]
> Mostly lgtm, but I just remembered: did we decide on keeping the (harvest) Year on the Settings level, or did we decide to move it to an intermediate entity, PLV style?

yes we did. but it's AFAIR part of @cristinamghita 's concept

**Comment by adi-stefan** [comment]
> > Mostly lgtm, but I just remembered: did we decide on keeping the (harvest) Year on the Settings level, or did we decide to move it to an intermediate entity, PLV style?
> 
> yes we did. but it's AFAIR part of @cristinamghita 's concept

Then just removing the interface classes and it will lgtm.

**Comment by cristinamghita** [comment]
> > Mostly lgtm, but I just remembered: did we decide on keeping the (harvest) Year on the Settings level, or did we decide to move it to an intermediate entity, PLV style?
> 
> yes we did. but it's AFAIR part of @cristinamghita 's concept

@metas-ts (cc @adi-stefan ) yes, we decide it move it.
Is one of my ACs.
See AC3 here: https://github.com/metasfresh/me03/issues/16108#issuecomment-1600621986

**Review by adi-stefan — APPROVED** [review]

**Pull request merged**